services/cannon/Makefile

```diff
@@ -6,6 +6,7 @@ BUILD_LABEL  ?= local
 BUILD        := $(BUILD_NUMBER)$(shell [ "${BUILD_LABEL}" == "" ] && echo "" || echo ".${BUILD_LABEL}")
 DEB          := $(NAME)_$(VERSION)+$(BUILD)_amd64.deb
 EXECUTABLES  := $(NAME)
+EXE_IT		 := dist/$(NAME)
```

Comment:
`cannon` doesn't actually have integration tests, only unit tests, which run with `stack build --test`. The generated executable, if there was one, would be called `cannon-integration`, not just `$(NAME)`. Remove this line again?